### PR TITLE
Make calibrate chart selector univariable

### LINKED ISSUE #4229: [TASK]: Make calibrate chart selector univariable
**Describe the task**
The chart can only display one variable at once, but the dropdown allow the user to select multiple.

### PR DESCRIPTION
**Pull request by YohannParis**
# Description

* Rename `tera-calibrate-ciemss.vue` to `tera-calibrate-ciemss-drilldown.vue`
* Clean up
* Add a prop `multiSelect` to decide betwen a dropdwon and a multiselect

Resolves #4229

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue [hunk 1]
```diff
@@ -90,19 +90,21 @@
...
 
 				<!-- Variable charts -->
 				<div v-if="!showSpinner" class="form-section">
 					<h5>Variables</h5>
 					<section v-if="modelConfig && csvAsset" ref="outputPanel">
 						<template v-for="(cfg, index) of node.state.chartConfigs" :key="index">
 							<tera-chart-control
-								:variables="Object.keys(pyciemssMap)"
 								:chartConfig="{ selectedRun: selectedRunId, selectedVariable: cfg }"
+								:multiSelect="false"
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue:105]: make props kebab-case

**Review by shawnyama — APPROVED** [review]
lgtm

File: packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue (hunk 1 above)
Comment (YohannParis) [reply packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue]: Nice catch, done!

**Review by mwdchang — APPROVED** [review]
looks okay

**Pull request merged**